packages/backend/src/handlers/triggers/playerRankUpdated.ts

```diff
@@ -104,6 +106,15 @@ export const playerRankUpdated = async (payload: TriggerPayload<Player>) => {
       console.warn(`Discord role associated with ${newRank} was not found!`);
     } else {
       await discordPlayer.roles.add([discordRoleForRank]);
+      // We should check whether this role is higher than the previous
+      if (playerRankedUp(oldPlayer?.rank, newPlayer?.rank)) {
+        const propsChannel = discordClient.channels.cache.get(
+          Constants.DISCORD_PROPS_CHANNEL_ID,
+        ) as TextChannel;
+        propsChannel.send(
+          `Props to ${newPlayer?.username} for becoming ${newRank}, congrats!`,
```

Comment:
Could we tag the user here instead of printing just their username?

Reply:
Good one, we should. I'll add it thanks!